hastexo/hastexo.py

```diff
@@ -43,14 +47,18 @@ def __init__(self, error_msg):
 @XBlock.wants('settings')
 class HastexoXBlock(XBlock,
                     XBlockWithSettingsMixin,
-                    StudioEditableXBlockMixin):
+                    StudioEditableXBlockMixin,
+                    StudioContainerWithNestedXBlocksMixin):
     """
     Provides lab environments and an SSH connection to them.
 
     """
+    CATEGORY = "hastexo"
+    STUDIO_LABEL = "hastexo XBlock"
+
     # Settings with defaults.
     display_name = String(
-        default="Lab",
+        default="hastexo XBlock",
```

Comment:
Does this mean that all courses using this component would need to be updated so they override this new default to keep saying `Lab`?

Reply:
Yes, but note that this is only really visible in Studio.  In the LMS, the display name of individual blocks is invisible.

I changed it because saying "Lab" is less than descriptive, specially when picking from the list of advanced problems in Studio.

Reply:
OK, makes sense.